src/preamble_minimal.js

```diff
@@ -116,6 +85,10 @@ else {
 #endif // MODULARIZE
 #endif // PTHREADS
 
+#if ASSERTIONS && SHARED_MEMORY
```

Comment:
Unrelated change?

Reply:
Actually this is an assert from the `MINIMAL_RUNTIME` version of the `updateMemoryViews` function.  I'm not sure we really need to preserve it but I decided to keep it around to be on the safe side.